clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/operations/ChainResolver.java

```diff
@@ -44,24 +53,50 @@ public ChainResolver(final OperationsCodec<K, V> codec) {
    * @param key a key
    * @return an entry with the resolved operation for the provided key as the key
    * and the compacted chain as the value
-   * @throws ClassNotFoundException
    */
-  public ResolvedChain<K, V> resolve(Chain chain, K key) {
+  public ResolvedChain<K, V> resolve(Chain chain, K key, long now) {
```

Comment:
You may wanna declare this additional parameter in the javadoc above. And as I read the javadoc for the return, it is wrong. My bad. I forgot to correct it when I changed the return type. While you're at it could you correct that too, please.